data/orders.yml

```diff
@@ -217,8 +217,8 @@
   office: Forest Service
   rfq_id: null
   title: Forest Service e-Permitting
-  solicitation_date: 2018-04-15
+  solicitation_date: TBD
```

Comment:
this TBD breaks the sort because it is looking for a date, so I suggest we either put something or remove it.